lib/ansible/modules/template.py

```diff
@@ -14,7 +14,7 @@
 ---
 module: template
 version_added: historical
```

Comment:
I don't think I'd seen `version_added: historical` before. It's awesome.

Reply:
i could change it to `fossil`, imo, would be also good